dev/gqltest/search_test.go

```diff
@@ -717,7 +717,7 @@ func testSearchClient(t *testing.T, client searchClient) {
 				zeroResult: true,
 				wantAlert: &gqltestutil.SearchAlert{
 					Title:       "No results",
-					Description: "It looks like you may have meant to run a structural search, but it is not toggled.",
+					Description: "It looks like you may have meant to run a structural search, but the patterntype is not specified or it is not toggled.",
```

Comment:
```suggestion
					Description: "It looks like you're trying to run a structural search, but it was not enabled using the patterntype keyword or UI toggle.",
```

Being clear about a search keyword vs a UI element might make it easier and/or quicker for folks to figure out how to resolve the issue. Without saying what the patterntype value is, a user would still need to look up docs, but being able to look for "keyword patterntype" might more quickly find the solution.

Are there any copy guidelines for whether to communicate the exact resolution to a user? For example, adding on: "Please enable structural search and try again." I know I've seen general copy guidelines, but don't recall seeing anything around error messages specifically (but I'm also still learning and could have missed something!). Not sure if design would have input here. :)

Reply:
Thanks for the better words, they're not my strong suit. I haven't seen any guidelines before, but I can ask tomorrow

Reply:
I usually default to asking design to proof read user-facing copy to make sure it has the seal of approval to fit in with the rest of the copy in the app they've put together. It's really hard to pick words sometimes 😅